aiven/client/cli.py

```diff
@@ -1298,6 +1298,147 @@ def service__connector__restart_task(self):
         project_name = self.get_project()
         self.client.restart_kafka_connector_task(project_name, self.args.name, self.args.connector, self.args.task)
 
+    @arg.project
+    @arg.service_name
+    @arg("schema-id", help="ID of a schema")
+    def service__schema__get(self):
+        """Get Kafka Schema Registry schema"""
+        project_name = self.get_project()
+        schema = self.client.get_schema(project_name, self.args.name, getattr(self.args, "schema-id"))
+        self.print_response(schema)
+
+    @arg.project
+    @arg.service_name
+    @arg.subject
+    @arg.schema
+    def service__schema__create(self):
+        """Create Kafka Schema Registry schema"""
+        project_name = self.get_project()
+        schema = self.client.create_schema_subject_version(project_name, self.args.name, self.args.subject, self.args.schema)
+        self.print_response(schema)
+
+    @arg.project
+    @arg.service_name
+    @arg.subject
+    @arg.version_id
+    @arg.schema
+    def service__schema__check(self):
+        """Check Kafka Schema Registry schema compatibility"""
+        project_name = self.get_project()
+        schema = self.client.check_schema_compatibility(project_name,
```

Comment:
```
        schema = self.client.check_schema_compatibility(
            project_name,
            self.args.name,
            self.args.subject,
            self.args.version_id,
            self.args.schema,
        )
```

and
```
        schema = self.client.check_schema_compatibility(
            project_name, self.args.name, self.args.subject, self.args.version_id, self.args.schema
        )
```

are the options for splitting into multiple lines that should be used (even though this project doesn't have static checker to enforce that and the style isn't currently consistently applied everywhere). Same applies to other similar cases.

Reply:
ok. noted